draft-bradleylundberg-arkg.md

```diff
@@ -498,7 +510,7 @@ TODO: IANA registry? COSE/JOSE?
 ### Using elliptic curve arithmetic for key blinding
 
 Instantiations of ARKG whose output keys are elliptic curve keys
-can use elliptic curve arithmetic as the key blinding scheme `BL`. [Frymann] [Wilson]
+can use elliptic curve arithmetic as the key blinding scheme `BL`. [Frymann2020]
```

Comment:
[Frymann] to [Frymann2020] is correct, thanks. But should we keep the [Wilson] reference here?

Reply:
done